qualtrics_link/management/commands/qualtrics.py

```diff
@@ -0,0 +1,454 @@
+import json
+import logging
+
+from django.core.management.base import BaseCommand
+
+from icommons_common.models import Person
+from qualtrics_link import util
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'This command works in three steps; ' \
+           '1. Get all Qualtrics users; ' \
+           '2. Find those that need to be updated;' \
+           '3. Update the users from step 2 via the Qualtrics API.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--get-users', action='store_true', help='Query Qualtrics to get all users and store them '
+                                                                     'in a data.json file.')
+        parser.add_argument('--filter-users', action='store_true', help='Filters the data.json file to produce a new'
+                                                                        'file containing only users that need to be '
+                                                                        'updated.')
+        parser.add_argument('--amount', default=50, type=int, help='The amount of users to filter in a run.')
```

Comment:
Maybe adding a sample line showing 'filter-users' command with the options may be helpful(I know it's in the wiki, but for those that may read the code and run it, would be useful)